catalog/app/containers/Auth/SSOAzure.js

```diff
@@ -0,0 +1,110 @@
+import { push } from 'connected-react-router/esm/immutable'
+import invariant from 'invariant'
+import * as React from 'react'
+import { FormattedMessage as FM } from 'react-intl'
+import * as redux from 'react-redux'
+import * as M from '@material-ui/core'
+
+import { useIntl } from 'containers/LanguageProvider'
+import * as Notifications from 'containers/Notifications'
+import * as Config from 'utils/Config'
+import * as NamedRoutes from 'utils/NamedRoutes'
+import * as Azure from 'utils/Azure'
+import * as Sentry from 'utils/Sentry'
+import defer from 'utils/defer'
+
+import * as actions from './actions'
+import * as errors from './errors'
+import msg from './messages'
+
+import microsoftLogo from './microsoft-logo.svg'
+
+const MUTEX_POPUP = 'sso:azure:popup'
+const MUTEX_REQUEST = 'sso:azure:request'
+
+export default function SSOAzure({ mutex, next, ...props }) {
+  const cfg = Config.useConfig()
+  invariant(!!cfg.azureClientId, 'Auth.SSO.Azure: config missing "azureClientId"')
+  invariant(!!cfg.azureTenantId, 'Auth.SSO.Azure: config missing "azureTenantId"')
+  const authenticate = Azure.use({
+    clientId: cfg.azureClientId,
+    authUrl: `https://login.microsoftonline.com/${cfg.azureTenantId}/oauth2/v2.0/authorize`,
```

Comment:
i think we should pass `tenantId` as a parameter and construct the url inside `Azure.use`